pages/browser-support.html

```diff
@@ -4,37 +4,27 @@
 
 <h2>Current Active Support</h2>
 
-<table id="support-matrix">
-	<thead>
-		<tr>
-			<th></th>
-			<th>Internet Explorer</th>
-			<th>Edge, Opera, Yandex.Browser</th>
-			<th>Chrome, Firefox, Safari</th>
-			<th>iOS</th>
-			<th>Android</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<th>jQuery</th>
-			<td>9+</td>
-			<td rowspan="2">Current</td>
-			<td rowspan="2">(Current - 1) and Current</td>
-			<td rowspan="2">7+</td>
-			<td rowspan="2">2.3, 4.0+</td>
-		</tr>
-		<tr>
-			<th>jQuery Compat</th>
-			<td>8+</td>
-		</tr>
-	</tbody>
-</table>
+<ul>
+	<li>Internet Explorer: 9+</li>
+	<li>Chrome: (Current - 1) and Current</li>
+	<li>Edge: Current</li>
+	<li>Firefox: (Current - 1) and Current</li>
+	<li>Safari: (Current - 1) and Current</li>
+	<li>Android: 4.0+ (plus limited 2.3 support)</li>
+	<li>iOS: 7+</li>
+	<li>Opera: Current</li>
+	<li>Yandex.Browser: Current</li>
+</ul>
 
 <p>Any problem with jQuery in the above browsers should be reported as a bug in jQuery.</p>
+
 <p><em>(Current - 1) and Current</em> denotes that we support the current stable version of the browser and the version that preceded it. For example, if the current version of a browser is 24.x, we support the 24.x and 23.x versions.</p>
+
 <p><em>Current</em> denotes that we support only the latest stable build of the browser.</p>
-<p>If you need to support older browsers like Internet Explorer 6-7, Opera 12.17 or Safari 5.1-6.0, use <a href="https://code.jquery.com/jquery-1.11.3.min.js" download>jQuery 1.11.3</a>.</p>
+
+<p>Android 2.3 is supported in a very limited way: jQuery shouldn't crash on it and basic methods should work; edge cases &amp; various bugs are not worked around.</p>
+
+<p>If you need to support older browsers like Internet Explorer 6-7, Opera 12.16-12.17 or Safari 5.1+, use <a href="https://code.jquery.com/jquery-1.11.3.min.js" download>jQuery 1.11.3</a>.</p>
```

Comment:
> Internet Explorer 6-8, right?

True, I'll correct that in a minute.

> we might as well drop the Opera minor versions in favor of just "Opera 12".

Opera 12 suggests Opera 12.0 and we definitely don't support that. Opera used to have a really weird versioning strategy and the upgrade from 12.0 to 12.10 changed a lot. Only then regular patches followed, up until 12.16 for Linux & OS X and 12.17 for Windows.

This line used to say 12.17 but since it's Windows-only I wanted to mention 12.16 in a way as well.